python/lib/dcoscli/tests/integrations/test_sanity.py

```diff
@@ -78,19 +78,6 @@ def test_repo_list():
         ['dcos', 'package', 'repo', 'list', '--json'], stdout=repo_list)
 
 
-def test_package_describe():
```

Comment:
Removed this test because it duplicates https://github.com/dcos/dcos-core-cli/pull/382/files#diff-f8ddb21a8ba9f7855ba87efde3fbd102R194